packages/app-mobile/components/ScreenHeader/Menu.tsx

```diff
@@ -105,42 +107,54 @@ const MenuComponent: React.FC<Props> = props => {
 		}
 	}
 
+	const [open, setOpen] = useState(false);
+
 	const onMenuItemSelect = useCallback((value: unknown) => {
 		if (typeof value === 'function') {
 			value();
 		}
 		setRefocusCounter(undefined);
+		setOpen(false);
 	}, []);
 
 	// debounce: If the menu is focused during its transition animation, it briefly
 	// appears to be in the wrong place. As such, add a brief delay before focusing.
-	const onMenuOpen = useMemo(() => debounce(() => {
+	const onMenuOpened = useMemo(() => debounce(() => {
 		setRefocusCounter(counter => (counter ?? 0) + 1);
+		setOpen(true);
 	}, 200), []);
 
 	// Resetting the refocus counter to undefined causes the menu to not be focused immediately
 	// after opening.
-	const onMenuClose = useCallback(() => {
+	const onMenuClosed = useCallback(() => {
 		setRefocusCounter(undefined);
+		setOpen(false);
+	}, []);
+
+	const menuRef = useRef<Menu|null>(null);
+	const onRequestMenuClose = useCallback(() => {
+		void menuRef.current.close();
 	}, []);
 
 	return (
 		<Menu
+			ref={menuRef}
 			onSelect={onMenuItemSelect}
-			onClose={onMenuClose}
-			onOpen={onMenuOpen}
+			onClose={onMenuClosed}
+			onOpen={onMenuOpened}
 			style={styles.contextMenu}
 		>
 			<MenuTrigger style={styles.contextMenuButton} testID='screen-header-menu-trigger'>
 				{props.children}
 			</MenuTrigger>
 			<MenuOptions>
-				<ScrollView
-					style={styles.menuContentScroller}
-					aria-modal={true}
-					accessibilityViewIsModal={true}
-					testID={`menu-content-${refocusCounter ? 'refocusing' : ''}`}
-				>{menuOptionComponents}</ScrollView>
+				<FocusControl.ModalContent visible={open}>
```

Comment:
It may make sense to document how focus management works in a technical spec. Such a technical spec might include:
- What `FocusControl.ModalContent` is and how it affects other parts of the application.
- What `FocusControl.MainAppContent` is and how it affects focusability.
- How focusing `AccessibleView`s works and how this interacts with `FocusControl.ModalContent`.

Reply:
Done in https://github.com/laurent22/joplin/pull/11929/commits/7ffa4c9a6b4106397c9f8fc00cc6e9cbbabf37ba.